tripal/config/install/views.view.tripal_content_type_listing.yml

```
@@ -0,0 +1,617 @@
+          items_per_page: 25
+          total_pages: null
+          id: 0
+          tags:
+            next: ››
+            previous: ‹‹
+            first: '« First'
+            last: 'Last »'
+          expose:
+            items_per_page: false
+            items_per_page_label: 'Items per page'
+            items_per_page_options: '5, 10, 25, 50'
+            items_per_page_options_all: false
+            items_per_page_options_all_label: '- All -'
+            offset: false
+            offset_label: Offset
+          quantity: 9
+      exposed_form:
+        type: basic
+        options:
+          submit_button: Apply
+          reset_button: false
+          reset_button_label: Reset
+          exposed_sorts_label: 'Sort by'
+          expose_sort_order: true
+          sort_asc_label: Asc
+          sort_desc_label: Desc
+      access:
+        type: none
+        options: {  }
+      cache:
+        type: none
+        options: {  }
+      empty: {  }
+      sorts:
+        created:
+          id: created
+          table: tripal_entity
+          field: created
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: tripal_entity
+          entity_field: created
+          plugin_id: date
+          order: DESC
+          expose:
+            label: Created
+            field_identifier: created
+          exposed: true
+          granularity: second
+        title:
+          id: title
+          table: tripal_entity
+          field: title
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: tripal_entity
+          entity_field: title
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: Title
+            field_identifier: title
+          exposed: true
+        type:
+          id: type
+          table: tripal_entity
+          field: type
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: tripal_entity
+          entity_field: type
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: Type
+            field_identifier: type
+          exposed: true
+        uid:
+          id: uid
+          table: tripal_entity
+          field: uid
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: tripal_entity
+          entity_field: uid
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: Authors
+            field_identifier: uid
+          exposed: true
+      arguments: {  }
+      filters:
+        title:
+          id: title
+          table: tripal_entity
+          field: title
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: tripal_entity
+          entity_field: title
+          plugin_id: string
+          operator: '='
```

Allows the title filter to match any part of the title.

```suggestion
          operator: 'contains'
```